libs/common-utils/src/validation/isValidInteger.ts

```diff
@@ -0,0 +1,7 @@
+export function isValidInteger(min: number, max: number) {
```

Comment:
Nitpick: at first I wasn't sure why this fn was returning a fn. Maybe use a name that reflects it better, like: `isValidIntegerFactory` or something similar, to indicate this meant to be the source of another fn.